src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ballRandom = new Ball();
+        ballRandom.setNumber(new Random().nextInt(100));
+        ballRandom.setColor(new ColorSupplier().getRandomColor());

```

Comment:
Make Random and ColorSupplier consant variables of class.
Also, better to use SecureRandom implementation.